scripts/release/controller/Jenkinsfile

```diff
@@ -0,0 +1,106 @@
+def USER_INPUT
+def S3_PREFIX = "s3://algorand-dev-deb-repo/releases"
+
+pipeline {
+    environment {
+        AWS_ACCESS_KEY_ID = credentials("aws-access-key-id")
+        AWS_SECRET_ACCESS_KEY = credentials("aws-secret-access-key")
+    }
+
+    agent any
+
+    stages {
+        stage("create ec2 instance") {
+            steps {
+                // Ubuntu Server 18.04 LTS, 8 vCPUs, 32 GB RAM
+                sh script: 'scripts/release/start_ec2_instance.sh us-west-1 ami-0dd655843c87b6930 t2.2xlarge'
+                //sh script: 'scripts/release/start_ec2_instance.sh us-east-1 ami-04b9e92b5572fa0d1 t2.2xlarge'
+            }
+        }
+
+        stage("setup ec2 instance") {
+            steps {
+                sh 'aws s3 cp s3://algorand-devops-misc/tools/gnupg2.2.9_centos7_amd64.tar.bz2 .'
+                sh 'ssh -i ReleaseBuildInstanceKey.pem -A ubuntu@$(cat scripts/release/tmp/instance) mkdir docker_test_resources'
+                sh 'scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r gnupg2.2.9_centos7_amd64.tar.bz2 ubuntu@$(cat scripts/release/tmp/instance):~/docker_test_resources/'
+                sh 'scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r scripts/release/controller/setup.sh ubuntu@$(cat scripts/release/tmp/instance):~/setup.sh'
+
+                script {
+                    USER_INPUT = input(
+                        id: 'user_inputs',
+                        message: 'Enter the Git tag or branch and the channel',
+                        parameters: [
+                            [
+                                $class: 'StringParameterDefinition',
+                                defaultValue: '',
+                                description: 'Git tag or branch',
+                                name: 'tag'
+                            ],
+                            [
+                                $class: 'StringParameterDefinition',
+                                defaultValue: 'stable',
+                                description: 'channel',
+                                name: 'channel'
+                            ]
+                        ]
+                    )
```

Comment:
These inputs look like things that you would know ahead of time, you can define task-level parameters instead of runtime parameters:
https://github.com/algorand/go-algorand-ci/blob/master/jenkins-tasks/release/Jenkinsfile#L4

Reply:
I think that's a good idea if we had a frontend to the jenkins server, but if I kick off a build from the jenkins dashboard then I like the idea of being prompted for the params rather than having to hardcode them in the config beforehand.

Reply:
Ah, I see, nevermind my last comment.  Yes, that will work well, thanks!